packages/io/amf-deserializer/deserialize.js

```diff
@@ -3,7 +3,7 @@ const createObject = require('./objectBuilder')
 
 const parse = require('./parse')
 
-const instantiate = (src, filename, options) => {
+const instantiate = (src, options) => {
```

Comment:
should it not be the other way around ? ie options, src to match the rest of the api ?

Reply:
(I am completely ok with keeping it as it is so far, so that only the wrapper function works that way)

Reply:
OK. I’ll change. Please hold off on the merge.

Reply:
Done